koku/masu/database/sql/openshift/cost_model/distribute_platform_cost.sql

```diff
@@ -118,14 +118,22 @@ SELECT
     NULL as persistentvolumeclaim_usage_gigabyte_months,
     UUID '{{source_uuid | sqlsafe}}' as source_uuid,
     'platform_distributed' as cost_model_rate_type,
-    CASE
-        WHEN {{distribution}} = 'cpu' AND (cost_category_id IS NULL OR max(cat.name) != 'Platform')
-            THEN sum(pod_effective_usage_cpu_core_hours) / max(udps.usage_cpu_sum) * max(pc.platform_cost)::decimal
-        WHEN {{distribution}} = 'memory' AND (cost_category_id IS NULL OR max(cat.name) != 'Platform')
-            THEN sum(pod_effective_usage_memory_gigabyte_hours) / max(udps.usage_memory_sum) * max(pc.platform_cost)::decimal
-        WHEN max(cat.name) = 'Platform'
-            THEN 0 - SUM(COALESCE(infrastructure_raw_cost, 0) +
-                COALESCE(infrastructure_markup_cost, 0)+
+    CASE WHEN {{distribution}} = 'cpu' AND (cost_category_id IS NULL OR max(cat.name) != 'Platform') THEN
+        CASE WHEN max(udps.usage_cpu_sum) = 0 THEN
```

Comment:
I think these should be `<=` because a negative usage is literal nonsense.
```suggestion
        CASE WHEN max(udps.usage_cpu_sum) <= 0 THEN
```

Reply:
`<=` it shall be! Updated.